packages/ruleset/src/utils/logger-factory.js

```diff
@@ -44,6 +44,7 @@ module.exports = class LoggerFactory {
       loggerName: name,
       logLevel
     });
+    this.applySettingsToAllLoggers();
```

Comment:
Why do we need to call this each time we add a new setting? Could we not just call `applyLoggerSettings`? (I might be misunderstanding the setup of this class, it just seems like this could make a lot of redundant function calls)

Reply:
discussed on this morning's standup... There are "reasons" why we do this, but I will take a closer look to see if we can streamline this logger settings stuff any more.